### ISSUE TITLE
[FR] Auto-pickup all?

### ISSUE THREAD

**Issue by x02Sylvie** [Nordgaren/Erd-Tools-CPP#5]
Hello,

I am having issue with auto-pickup feature in modded gameplay (Only mod I use is Vigil and Erd Tools CPP)
https://www.nexusmods.com/eldenring/mods/2832

My settings are:
[FRONTEND]
; Auto pickup items when in range     
pickup_materials = 1
pickup_items = 1
pickup_corpse_loot = 1
pickup_lost_runes = 0

I am unsure of exact cause but auto-pickup does not seem to work for everything at the moment with Vigil mod. For example it does not pick up mushrooms, rowa fruit, and another stuff, so i have to manually press E to pickup

Would it be possible to include something like 'pickup_all' to resolve such issue?
- Thanks

**Comment by Nordgaren** [comment]
> Hello,
> 
> I am having issue with auto-pickup feature in modded gameplay (Only mod I use is Vigil and Erd Tools CPP) https://www.nexusmods.com/eldenring/mods/2832
> 
> My settings are: [FRONTEND] ; Auto pickup items when in range pickup_materials = 1 pickup_items = 1 pickup_corpse_loot = 1 pickup_lost_runes = 0
> 
> I am unsure of exact cause but auto-pickup does not seem to work for everything at the moment with Vigil mod. For example it does not pick up mushrooms, rowa fruit, and another stuff, so i have to manually press E to pickup
> 
> Would it be possible to include something like 'pickup_all' to resolve such issue?
> 
>     * Thanks

Does this change if you don't have Vigil installed?

**Comment by x02Sylvie** [comment]
Yeah, I had no issues without Vigil mod. No issues when I played with Elden Ring Reforged (ERR) mod either.

So I assume Vigil mod tinkers with something related to things like rowa fruits and mushrooms. Since for example automatically picking up items from dead enemies still works fine in vigil but not stuff like mushrooms, rowa fruit,

**Comment by Nordgaren** [comment]
Good news:

Vawser told me what it was, and I can fix it

Bad news:
IDK when. Hopefully next week. Ty for the bug report!
